ui/app/components/calendar-widget.js

```diff
@@ -12,8 +12,10 @@ import { tracked } from '@glimmer/tracking';
  * ```js
  * <CalendarWidget
  * @param {string} endTimeDisplay - The display value of the endTime. Ex: January 2022.
+ * @param {string} endTimeAtInit - Because endTimeDisplay is a tracked property of the parent it changes. But we need to keep a value unchanged that records the endTime returned by the counters/activity endpoint. This is that value.
```

Comment:
will this always be the endTime from the response right? I think we had talked about calling this `endTimeFromResponse` or something - but maybe I'm misremembering.